src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+
+        Figure[] figures = {supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure()
```

Comment:
Instead of creating a new 'FigureSupplier' instance inside the loop, it's better to do it once before the loop starts to avoid unnecessary object creation.